client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -80,12 +80,16 @@ function SudoModePasswordField(props) {
    */
   function renderConfirm() {
     const helpLink = clientConfig.helpLink;
+    let { verifyMessage } = props;
```

Comment:
At the top of the component onSuccess and autocomplete props are assigned to variables, so the same with this prop, and use a different variable for verifyMessage below

Reply:
> At the top of the component onSuccess and autocomplete props are assigned to variables, do the same with this prop

... It.... *is* assigned to a variable? What do you mean?

Reply:
Reading again this morning... I *think* you mean you want this variable assignment to be done in the same place as the other two? That's a very subjective change, which to my eye is not as good as what I've done, since this is only used right here, not elsewhere in the file.
But in the interest of unlocking the beta, I'll make that change.

Reply:
Done. I don't like having two variables for the price of one, and I couldn't think of a good name for the new variable because the name of the variable should be `verifyMessage` - but I've made the requested change.